VideoWeb/VideoWeb/ClientApp/src/app/services/clients/api-client.ts

```diff
@@ -2481,7 +2482,7 @@ export interface ITestCallScoreResponse {
 
 export class UpdateParticipantStatusEventRequest implements IUpdateParticipantStatusEventRequest {
     participant_id?: string;
-    readonly event_type?: EventType;
```

Comment:
this property should be readonly.  Same as previous reasons

Reply:
I think we could reuse this class

Reply:
Resolved as above by re-using this method with an updated reason.